src/applications/resources-and-support/components/SearchResultList.jsx

```diff
@@ -16,51 +16,58 @@ const articleTypes = {
+      />
+      {hasTopics && (
+        <div className="vads-u-margin-top--1">
+          <dfn className="vads-u-font-weight--bold">Topics </dfn>
+          {article.fieldPrimaryCategory?.entity?.entityUrl && (
+            <Tag term={article.fieldPrimaryCategory?.entity} />
+          )}
+          {article.fieldOtherCategories?.map((otherCategory, index) => {
+            return (
+              <Tag key={`category-${index}`} term={otherCategory.entity} />
+            );
+          })}
+        </div>
+      )}
+      {article.fieldTags?.entity?.fieldAudienceBeneficiares?.entity?.name && (
+        <div className="vads-u-margin-top--1">
+          <dfn className="vads-u-font-weight--bold">Audience </dfn>
+          <Tag
+            term={article.fieldTags.entity.fieldAudienceBeneficiares.entity}
+          />
+        </div>
+      )}
+    </div>
+  );
+}
+
 export default function SearchResultList({ results }) {
   return (
     <ul className="usa-unstyled-list">
       {results.map((article, articleIndex) => {
         return (
           <li key={`article-${articleIndex}`}>
-            <div className="vads-u-padding-y--3 vads-u-border-top--1px vads-u-border-color--gray-lighter">
-              <div>
-                <dfn className="sr-only">Article type: </dfn>
-                {articleTypes[article.entityBundle]}
-              </div>
-              <h2 className="vads-u-font-size--h3 vads-u-margin-top--0">
-                <a href={article.entityUrl.path}>{article.title}</a>
-              </h2>
-              <p
-                className="vads-u-margin-bottom--0"
-                // the article descriptions contain HTML entities
-                // eslint-disable-next-line react/no-danger
-                dangerouslySetInnerHTML={{ __html: article.description }}
-              />
-              <div className="vads-u-margin-top--1">
-                <dfn className="vads-u-font-weight--bold">Topics </dfn>
-                <Tag term={article.fieldPrimaryCategory?.entity} />
-                {article.fieldOtherCategories?.map((otherCategory, index) => {
-                  return (
-                    <Tag
-                      key={`category-${index}`}
-                      term={otherCategory.entity}
-                    />
-                  );
-                })}
-              </div>
-              {article.fieldTags?.entity?.fieldTopics?.length > 0 && (
-                <div className="vads-u-margin-top--1">
-                  <dfn className="vads-u-font-weight--bold">Audience </dfn>
-                  {article.fieldTags?.entity?.fieldTopics?.map(
-                    (fieldTopic, index) => {
-                      return (
-                        <Tag key={`topic-${index}`} term={fieldTopic.entity} />
-                      );
-                    },
-                  )}
-                </div>
-              )}
-            </div>
+            <SearchResult article={article} />
```

Comment:
The conditional rendering of the different fields was getting kinda wild so I separated out the search result into its own component.